README.rst

```diff
@@ -2,51 +2,122 @@
-* Make_: Used as single doorway to all your tasks:
-
-  * For setup (development environment and machine, CI/CD pipeline job environments)
-  * For test tasks
-  * For the whole development and release lifecycle
-
-* Tox_ testing: Set up to easily test for Python 3.6, 3.7, 3.8 and used as single source
-  for all your test invocations
-* Sphinx_ docs: Documentation ready for generation with, for example, `Read the Docs`_
-* GitLab-CI_: CI-CD pipeline with:
-
-  * Test phase running all tests in parallel jobs
-  * Release phase (see semantic-release below)
-
-* Semantic-release_: Pre-configured version bumping, changelog generation, auto-release
-  to PyPI_ and GitLab (yes... not a Python written tool... but works *really* well and
-  has way more advanced features than any of the existing Python tools)
-* Command line interface (optional) using Cleo_ (default), Click_ or Argparse_
+.. image:: https://github.com/opinionated-digital-center/python-library-project-generator/workflows/Test%20and%20make%20release/badge.svg
+    :target: https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+and+make+release%22
+
+.. image:: https://github.com/opinionated-digital-center/python-library-project-generator/workflows/Test%20GitLab%20Pipeline/badge.svg
+    :target: https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+GitLab+Pipeline%22
+
+* GitHub repo: https://github.com/opinionated-digital-center/python-library-project-generator .
+* Free software: MIT license.
+
+The "Opinionated Python Library Project Template" is a fully tested, feature rich
+`Cookiecutter`_ scaffold to kick start your Python library and command line interface
+development.
+
+It is "opinionated" as it reflects strong structural and tooling choices,
+which we believe help developers being more effective within Digital Centers. It can
+however seamlessly be used for Open Source development (works also out-of-the-box with
+gitlab.com, github.com, pypi.org and readthedocs.org, as a standalone project).
+
+In addition to the intrinsic features offered by the template (see "Features list"
+below), the specific features that make it suitable and effective for Digital Centers
+are the abilities to:
+
+* Use self-hosted GitLab and GitHub.
+* Publish to self-hosted package repositories.
+* Reuse centralised and optimised pipeline elements. (available soon)
+* Reuse other centralised "toolbox" items.
+
+The argumentation behind the Opinionated Digital Center's choices (for this and other
+projects) are/will be gradually documented as Architecture Decision Records, either
+on `ODC's ADR repository <https://github.com/opinionated-digital-center/architecture-decision-record>`_
+for transversal decisions, or on each individual repository's ``docs/adr`` directory
+(`here <https://github.com/opinionated-digital-center/python-library-project-generator/tree/master/docs/adr>`_
```

Comment:
Is it possible to use relative links to point to the ADRs directory of this branch ?

Reply:
done